go/tasks/plugins/k8s/sagemaker/constants.go

```diff
@@ -0,0 +1,45 @@
+package sagemaker
+
+import commonv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/common"
+
+const (
+	trainingJobTaskPluginID = "sagemaker_training"
+	trainingJobTaskType     = "sagemaker_training_job_task"
+)
+
+const (
+	hyperparameterTuningJobTaskPluginID = "sagemaker_hyperparameter_tuning"
+	hyperparameterTuningJobTaskType     = "sagemaker_hyperparameter_tuning_job_task"
+)
+
+const (
+	BayesianSageMakerAPIHyperParameterTuningJobStrategyType commonv1.HyperParameterTuningJobStrategyType = "Bayesian"
```

Comment:
It looks like we can just do `commonv1.HyperParameterTuningJobStrategyType(proto_enum.string())`
Is that not right?

Reply:
reason is you wont need a "switch" case.  thus if we add some new types, we just have to add it to the proto.